extensions/ql-vscode/src/model-editor/method-modeling/method-modeling-view-provider.ts

```diff
@@ -133,17 +133,27 @@ export class MethodModelingViewProvider extends AbstractWebviewViewProvider<
           this.databaseItem,
           msg.methodSignature,
         );
+
+        void telemetryListener?.sendUIInteraction(
```

Comment:
This will get triggered every time anyone changes any aspect of a modeling. Do we want it to trigger that frequently?

It will tell us that people are using the method modeling panel, but I wonder if we have other ways of telling that. Or do we want to get info at this level of detail and frequency?

Reply:
I've removed this - I was under the impression we did the same in the model editor, but we don't. If we find that we need information around this we can add some telemetry in the future.